### DISCUSSION TITLE
Getting weird "Doesn't look like _repository_ is a known repository" with a few user/organizations' repos

### DISCUSSION THREAD

**Discussion by mihirsamdarshi** [community/community#28015] [Other Feature Feedback, Questions, & Ideas]
I am trying to search cs.github.com and only search the code within [warp-tech/warpgate](https://cs.github.com/warp-tech/warpgate) and [warp-tech/russh](https://cs.github.com/warp-tech/russh). However, I get to an error page that says that the repository was not found. However, clearly the repository exists. 

Additional repos that exhibit this issue include [davenport-terra/tf_aws_ellucian_jobsub](https://cs.github.com/davenport-terra/tf_aws_ellucian_jobsub) and [ellipsis-ai/npm-ellipsis-aws-helper](https://cs.github.com/ellipsis-ai/npm-ellipsis-aws-helper)

I was going to suggest that perhaps it had to do with the fact that theses users all had dashes in their names, but I can pretty clearly query [fl-yu/CUT-RUNTools-2.0](https://cs.github.com/fl-yu/CUT-RUNTools-2.0), so I have no idea why this is happening

<img width="1186" alt="Screenshot 2022-08-02 at 12 59 25 AM" src="https://user-images.githubusercontent.com/5462077/182323405-d3207dd2-f199-4b6a-be4d-54397205a1d4.png">

**Comment by kimsey0** [comment]
I'm experiencing this with https://github.com/Hello1024/gse-shader (https://cs.github.com/Hello1024/gse-shader) right now, which doesn't have a hyphen in the username.
